Expo1900/Expo1900/Models/ExpoItemElement.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  ExpoItemElement.swift
+//  Expo1900
+//
+//  Created by 재재, 보리사랑 on 2022/06/13.
+//
+
+import Foundation
+
+// MARK: - ExpoItemElement
+struct ExpoItemElement: Codable {
+    let name, imageName, shortDesc, desc: String
```

Comment:
```suggestion
    let name: String
    let imageName: String
    let shortDesc: String
    let desc: String
```
=> 프로퍼티를 쉼표로 연달아 선언하는 것 보단 하나씩 선언하는게 파악하기도 쉽고 차후에 변경하기에도 좋다고 생각합니다. 
어떻게 생각하실지 궁금하네요 :)

Reply:
개인적으로 한 줄로 쓰면 보기 편하겠다 생각했었는데, 차후에 변경하기 좋고 오히려 하나씩 분리해서 선언하는게 파악하기 쉽다는 제이슨의 의견에 따라 변경해주었습니다. 감사합니다 !